receiver/redisreceiver/config.md

```diff
@@ -0,0 +1,267 @@
+# Redis Receiver Reference
+
+
+
+### redisreceiver-Config
+
+| Name | Field Info | Default | Docs |
+| ---- | --------- | ------- | ---- |
+| collection_interval |[time-Duration](#time-Duration)| 10s |  |
+| endpoint |string| <no value> |  |
+| transport |string| tcp |  |
+| password |string| <no value> | Optional password. Must match the password specified in the requirepass server configuration option.  |
+| tls |[tls-TLSClientSetting](#tls-TLSClientSetting)| <no value> |  |
+| metrics |[metrics-MetricsSettings](#metrics-MetricsSettings)| <no value> | MetricsSettings provides settings for redisreceiver metrics.  |
+
+### tls-TLSClientSetting
+
+| Name | Field Info | Default | Docs |
+| ---- | --------- | ------- | ---- |
+| ca_file |string| <no value> |  |
+| cert_file |string| <no value> |  |
+| key_file |string| <no value> |  |
+| min_version |string| <no value> |  |
+| max_version |string| <no value> |  |
+| reload_interval |[time-Duration](#time-Duration)| <no value> |  |
+| insecure |bool| true |  |
+| insecure_skip_verify |bool| <no value> |  |
```

Comment:
This should have a default, why doesn't it have one?

Reply:
Sorry, can you clarify? Are you suggesting that the Redis receiver should set a default value on `insecure_skip_verify` or that it already does and it's not showing up here?

I'm not seeing the receiver set a default value on this field, so the fact that we get <no value> in this markdown file is expected.

Reply:
The Redis receiver has a default for `insecure_skip_verify`: since the value is not explicitly set [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/979e0c1c90580fda6393e4867c3e0d873406cb03/receiver/redisreceiver/factory.go#L51-L53) the default zero value is used. The fact that is not explicitly set in the code is not relevant for end-users understanding what the receiver will do by default.

Reply:
Great catch @mx-psi. There was a bug in how booleans were handled. Should be fixed now.